docs/src/partials/java/_code-tabs-gbcontext-builder.mdx

```diff
@@ -0,0 +1,49 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+```mdx-code-block
+<Tabs>
+  <TabItem value="java" label="Java">
+```
+
+```java
+String featuresJson = featuresRepository.getFeaturesJson();
```

Comment:
I can't find anything about `featuresRepository` unless I dig into the examples source code.  Is this going to be clear to users?  Should we link to that example source code or add a comment to the code here?

Reply:
Updated to include the networking code we used in the Getting Started code tab.